app/src/test/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentLogicTest.kt

```diff
@@ -35,19 +36,19 @@ import org.simple.clinic.util.Just
 import org.simple.clinic.util.RxErrorsRule
 import org.simple.clinic.util.TestUserClock
 import org.simple.clinic.util.scheduler.TrampolineSchedulersProvider
-import org.simple.clinic.widgets.ScreenCreated
 import org.simple.clinic.widgets.UiEvent
 import org.simple.mobius.migration.MobiusTestFixture
 import org.threeten.bp.LocalDate
 import org.threeten.bp.Period
 import java.util.UUID
 
-class ScheduleAppointmentSheetControllerTest {
+class ScheduleAppointmentLogicTest {
```

Comment:
I know we don't really have rule that says we should do this, but can you add `given, when and then` comments in all tests in this class? They are present is some and missing in others.

Reply:
Sure, I'll make this change in the final clean up PR since I have already some changes to the tests there and I do not want to get into rebase conflicts.

Reply:
Addressed in https://github.com/simpledotorg/simple-android/pull/1341